src/pages/index.js

```diff
@@ -67,6 +67,8 @@ function handleAvatarSubmit(link) {
   function makeRequest() {
     return api.updateAvatar(link).then((newAvatar) => {
       userInfo.setUserInfo({
+        name: userInfo.getUserInfo().name,
+        about: userInfo.getUserInfo().about,
```

Comment:
`userInfo.getUserInfo()` is not needed here because you get the whole object from the server with all 4 params (including avatar)


```js
    return api.updateAvatar(link).then((userData) => {
      userInfo.setUserInfo(userData);
    });

```


Please, call it `userData` instead of `newAvatar` because the response has the whole data of the user

Reply:
Ah cheers! Thank you so much for your help here and the rest of my project 👍